Formula/recutils.rb

```diff
@@ -16,6 +16,13 @@ class Recutils < Formula
     sha256 "aec2464c5e26a561b340e9ae5a080366a068936ff2ba4e86e6a4bcf0ed8a95d0" => :mountain_lion
   end
 
+  if MacOS.version >= :high_sierra
```

Comment:
`recutils` compiles just fine currently, but most of the `rec*` binaries fail due to how `%n` is used. https://raw.githubusercontent.com/macports/macports-ports/b76d1e48dac/editors/nano/files/secure_snprintf.patch describes the issue and luckily works here as well.